### PR TITLE
add indexes for transcription model to aid with transcribot queue

### PR DESCRIPTION
**Pull request by itsthejoker**
Relevant issue: https://github.com/GrafeasGroup/blossom/pull/163

## Description:

Just adds a few indexes on the Transcription model to try and get times down.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [ ] Tests (if applicable)
- [x] Success Criteria Met
- [ ] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

File: api/models.py [hunk 1]
```diff
@@ -231,6 +231,9 @@ def save(self, *args: Any, **kwargs: Any) -> None:
 
 
 class Transcription(models.Model):
+    class Meta:
+        indexes = [models.Index(fields=["author", "submission", "original_id", "url"])]
```
Comment (TheLonelyGhost) [inline api/models.py:235]: Do you mean to create an index that's the combination of these fields? I'm not sure if that's helpful as much as, for example, indexing the `original_id` or the `url` fields on their own. Perhaps I'm misreading how Django interacts with the database, though.

Comment (itsthejoker) [reply api/models.py:235]: ...I don't know. Let me look into it.

Comment (itsthejoker) [reply api/models.py:235]: I think you're right. Will tweak.

Comment (TheLonelyGhost) [reply api/models.py]: In other words, I'm not sure you intended to create a [composite index](https://en.wikipedia.org/wiki/Database_index#Column_order) of these fields. If you did, I'm curious how that would improve search speed/efficiency.

**Pull request merged**